### PR TITLE
Feat/enhance quickstart

### PR DESCRIPTION
**Pull request by tiboun**
## Summary
Enhance and fix quickstart guide and make it buildable from windows

**Related Issue: #IssueId**
No one

**PR Type: Bug Fix | Documentation**

**Status: Ready to review**

**Breaking change? No**

### Solution
Change the way to load file in order to be compatible with windows, mainly during tests.
Add more explanation during quickstart.
Add missing file making quickstart bootstrapping impossible.

### How has this been tested?
Add unit tests
Replay quickstart guide with the local snapshot build

### Deploy Notes
none

## Contributor checklist:
Go over all the following points, and put an `x` in all the boxes that apply.
If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [x] My code follows the code style of this project.
- [x] I have updated the Release notes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/docs/quickstart/150.transform.mdx [hunk 1]
```diff
@@ -21,11 +21,14 @@ select * from customers where lower(country) like lower('{{ p_country }}')
...
-We need to create a YAML file that will instruct where the result of the SQL request will be stored.
+SQL file alone is not sufficient to be a Starlake Transform.
+In order to be one, a YAML file must be created next to the SQL file and have the same name.
+Therefore we need to create the YAML file `metadata/jobs/bycountry.comet.yml`.
+This file will instruct where the result of the SQL request will be stored.
 
 ```yaml
 transform:
-  name: bycountry
+  name: country-dispatch
```
Comment (hayssams) [inline docs/docs/quickstart/150.transform.mdx]: We need this name to the one used in the file name.

File: src/main/scala/ai/starlake/schema/handlers/LocalStorageHandler.scala [hunk 2]
```diff
@@ -147,7 +147,7 @@ class LocalStorageHandler(implicit
           else
             files.sortBy(f => (f.lastModifiedTime, f.name))
 
-        sorted.map(f => new Path(f.pathAsString))
+        sorted.map(f => new Path(f.uri))
```
Comment (hayssams) [inline src/main/scala/ai/starlake/schema/handlers/LocalStorageHandler.scala]: Does it make any difference ?

**Review by hayssams — CHANGES_REQUESTED** [review]
Gr8 PR. Lot of code quality and checks improvements.
Could you just make the name of the file and the name of the job the same in your example

File: docs/docs/quickstart/150.transform.mdx (hunk 1 above)
Comment (tiboun) [reply docs/docs/quickstart/150.transform.mdx]: Ok, without this it still works but probably with a warning that I didn't see. It still transform the data. This has been changed in order to highlight the note next to it `The `--name` will match the transform file name and not its name.`.

I can revert this change.

File: src/main/scala/ai/starlake/schema/handlers/LocalStorageHandler.scala (hunk 2 above)
Comment (tiboun) [reply src/main/scala/ai/starlake/schema/handlers/LocalStorageHandler.scala]: it just add the protocol prefix. Tried to restore pathAsString and fix failed unit tests. It works as well and build on windows. Change reverted.

**Review by hayssams — APPROVED** [review]
LGTM

**Pull request merged**